odoo/addons/base/models/res_company.py

```diff
@@ -326,6 +327,12 @@ def _unlink_if_company_has_no_children(self):
     def write(self, values):
         invalidation_fields = self.cache_invalidation_fields()
         asset_invalidation_fields = {'font', 'primary_color', 'secondary_color', 'external_report_layout_id'}
+
+        companies_needs_l10n = not getattr(threading.current_thread(), 'testing', False) and\
```

Comment:
Couldn't find any other better way to green the runbot (though security got triggered), I hope we are not planning it for the testcase :grimacing: 
@william-andre @jco-odoo